Automattic/wp-calypso
# State: Move inline help selectors

#### Changes proposed in this Pull Request

This PR moves all the selectors currently in `client/state/inline-help/selectors.js` to individual selector files in the global state directory in client/state/selectors, and update where these selectors are imported. Also, it moves and updates the currents tests.


#### Testing instructions

* Execute Unit tests for all global selectors.

```
> cd test/client
> yarn run test-client client/state/selectors/
```

* Check that either `Get Help` section in the customer home page is working as expected, as well as the `Inline Help Popover`. Check it records tracks events too.

* A visual code inspection is also a welcomed to test this PR


Fixes https://github.com/Automattic/wp-calypso/issues/42697
